services/checks/checks_test.go

```diff
@@ -156,6 +163,44 @@ uEF33ScMPYpvHvBKv8+yBkJ9k4+DCfV4nDs6kKYwGhalvkkqwWkyfJffO+KW7a1m3y42WHpOnzBxLJ+I
 	})
 }
 
+func TestStartChecks(t *testing.T) {
+	t.Run("stt disabled", func(t *testing.T) {
+		sqlDB := testdb.Open(t, models.SkipFixtures, nil)
+		db := reform.NewDB(sqlDB, postgresql.Dialect, nil)
+
+		defer func() {
+			require.NoError(t, sqlDB.Close())
+		}()
+
+		s := New(nil, nil, db, "2.5.0")
+		err := s.StartChecks(context.Background())
+		assert.EqualError(t, err, services.ErrSTTDisabled.Error())
+	})
+
+	t.Run("stt enabled", func(t *testing.T) {
+		sqlDB := testdb.Open(t, models.SkipFixtures, nil)
+		db := reform.NewDB(sqlDB, postgresql.Dialect, nil)
+
+		defer func() {
+			require.NoError(t, sqlDB.Close())
+		}()
+
+		var ar mockAlertRegistry
```

Comment:
Why we mock it?

Reply:
To not import one service package into another.